aksel.nav.no/website/pages/api/aksel-articles.ts

```diff
@@ -0,0 +1,47 @@
+import { akselArticleFields } from "@/lib";
+import { getClient } from "@/sanity-client";
+import { format } from "date-fns";
+import { NextApiRequest, NextApiResponse } from "next";
+
+export default async function akselAarticles(
+  req: NextApiRequest,
+  res: NextApiResponse
+) {
+  if (req.method !== "GET") {
+    return res.status(405).json({ message: "Method not allowed" });
+  }
+
+  let lastPublishedAt = req.query.lastPublishedAt;
+
+  if (!lastPublishedAt) {
+    return res
+      .status(400)
+      .json({ message: "Missing lastId or lstPublishedAt" });
+  }
+
+  lastPublishedAt = format(new Date(String(lastPublishedAt)), "yyyy-MM-dd");
+
+  const query = `{
+    "publishDateArtilcles": *[_type == "aksel_artikkel"  && defined(publishedAt) && publishedAt <= "${lastPublishedAt}"] | order(publishedAt desc) {
+          ${akselArticleFields}
+        },
+    "noPublishDateArticles": *[_type == "aksel_artikkel" && !defined(publishedAt)] | order(_updatedAt desc){
```

Comment:
Hva brukes denne til?

Reply:
Er en del artikler som ikke har "publishedAt", så er for å filtrere ut de å plassere de sist i feeden

Reply:
Siden vi bruker publishedAt til å bestemme rekkefølge